src/main/java/com/eternalcode/core/command/implementations/ItemNameCommand.java

```diff
@@ -33,7 +33,7 @@ public ItemNameCommand(final MessagesConfiguration messages, final PluginConfigu
 
 
     @Execute
-    @MinArgs(1)
+    @Required(1)
     public void execute(final Player player, @Arg(0) @Handler(StringPlayerArgument.class) final Option<String> stringOption) {
```

Comment:
`@Arg(0) @Handler(StringPlayerArgument.class) final Option<String> stringOption` - jeżeli chcesz gracza jako String
żeby tabowanie działało, a jak chcesz nazwę itemu to zamiast tego możesz dać adnotacje Joiner String text
działa to jak StringUtils.join args